invenio_sip2/datastore.py

```diff
@@ -81,8 +86,8 @@ def __init__(self, app=None):
         """Initialize the datastore."""
```

Comment:
Your parent `__init__()` function doesn't have the same signature. Here you use `app` as first argument, and `datastrore` for parent __init__. You should use "**kwargs" in both functions. Additionaly the override function should call the `super().__init__()` function.